Description: Fix wrong-path-for-interpreter (Perl interpreter)
 According Debian Policy Manual section 10.4 (Scripts) special exception, Perl
 scripts must use /usr/bin/perl directly and not /usr/bin/env.
Author: Sebastien Badia <sbadia@debian.org>
Forwarded: not-needed
Last-Update: 2018-08-26

--- a/utils/rspamd_stats.pl
+++ b/utils/rspamd_stats.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 use 5.010;
 use Data::Dumper;
